### DISCUSSION TITLE
Great blocklists. Do you have anything for vaping?

### DISCUSSION THREAD

**Discussion by PicoDoni** [blocklistproject/Lists#308] [Ideas]
Vaping is an interesting category, its not really tobacco, or drugs. I am looking for an up to date vaping blocklist. Do you have any plans for such a thing in the future?

**Comment by fishcharlie** [comment]
I think this is a good idea. It's all a matter of starting the list with some content, and going from there. @doni74 Do you have a list of starting domains you'd like to see for this list?

**Comment by blocklistproject** [comment]
At this time we do not have a list dedicated to vaping. Not at all opposed to the idea. Just wonder were would could get an initial list of domains...? Is this something that you might be interested in helping with?

**Reply by iam-py-test** [reply]
@blocklistproject you probably can find vaping domains just using search engines (DuckDuckGo, Bing, Google)

**Reply by fishcharlie** [reply]
@iam-py-test I think part of this is just finding the time to do it as well.

**Reply by iam-py-test** [reply]
> @iam-py-test I think part of this is just finding the time to do it as well.

I can find a few domains and open a PR if that would be easier.

**Reply by fishcharlie** [reply]
@iam-py-test All for that. Just do the following please:

- Label it `vaping.txt` in the root folder
- Add it to the README beta lists section
- Add it to the array as a commented out list (since it's beta)
- Don't generate the alternate lists

**Reply by iam-py-test** [reply]
@fishcharlie how do I prevent the alt lists from being created?
Do I just not add it to the array?

**Reply by fishcharlie** [reply]
@iam-py-test Just don't run any of the scripts. Just ensure your commit only adds that `vaping.txt` file, edits the README file, and updates the `create-everything-list.js` array (without running it).

**Reply by iam-py-test** [reply]
@fishcharlie @blocklistproject is https://github.com/blocklistproject/Lists/pull/384 look ok?

**Comment by client7676** [comment]
> Vaping is an interesting category, it's not really tobacco, or drugs. I am looking for an up-to-date vaping blocklist. Do you have any plans for such a thing in the future?
That's an interesting point! Vaping is indeed a unique category, and it's great that you're proactive about staying informed. While I don't have immediate plans for a vaping blocklist, I encourage you to explore and support initiatives that promote responsible and safe [vaping](https://grdnt.com/) practices. Keep an eye out for any updates from health organizations and authorities in this regard!
